### DISCUSSION TITLE
Nextjs 13 misunderstanding count of request per builded page

### DISCUSSION THREAD

**Discussion by atinseau** [vercel/next.js#43539] [Help]
... Im french sorry for my english... 

so i'm starting to using NEXT 13 and i'm completely lost about "new fetch process"

like the docs, i have a minimal example (basically a hello world), so page + layout blablabla
![image](https://user-images.githubusercontent.com/65445747/204613635-2e9c7752-2a6a-47f2-953a-c27635385c42.png)
![image](https://user-images.githubusercontent.com/65445747/204613694-669d51cd-7dc3-40e4-8540-986baf93b457.png)

my page file request a custom service (pocketbase but whatever) and is cached like the docs expects !
(with console.log to see how many time the function is called)

![image](https://user-images.githubusercontent.com/65445747/204613914-f95fb012-a235-47a8-aaa4-1824e05704f8.png)

but during the build, this is happening...
![image](https://user-images.githubusercontent.com/65445747/204614117-58ece247-adfd-4cd3-89de-625f8f4623a2.png)

Maybe i didn't understand the new cache/fetch system, i need help haha

Thank you !

**Comment by utg1-tdawg** [comment]
Specifically, are you asking why is `fetch` called twice during build? I noticed this as well. But I am still unable to find out why. There is a similar [unanswered question](https://stackoverflow.com/questions/74486396/why-is-fetch-called-twice-during-build-in-nextjs-13) on stackoverflow.
